helm/README.md

```diff
@@ -0,0 +1,277 @@
+# zaakafhandelcomponent
```

Comment:
Does this file need to be maintained manually or how does this work? Updating it manually will be quite a challenge.. e.g. when new params are added or removed from our Helm Chart.  

Also this overlaps in part with the env var descriptions we have in our `.env.example` file. Ideally we should have one place where we describe our ZAC env vars / Helm Chart vars, although the use cases do differ I guess since  not every ZAC user will even use the Helm Chart..

Reply:
It's semi-automatic for now. I just run: `helm-docs helm/values.yml` to get this to work and generate the `README`.

Was looking for an easy way to automate this, but no gradle plugin for it, so we have to craft something that downloads the appropriate binary (OS/architecture), or go with `go get`. Would add some complexity to the build and story points to the spring. Perhaps for a separate story?

Reply:
On the other side we have https://github.com/marketplace/actions/helm-docs-github-action

Reply:
I'd try to integrate that github action 👍

Reply:
The github action means additional push and build without the ability to review what's the output.

I went for helm-docs local execution in the PR.